src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Record.txt

```diff
@@ -15,8 +15,8 @@ false
 >> IsBlank({ x : Blank(), y : 2}.x)
 true
 
->> First([{a: 1}]).Value.a
+>> First([{a: 1}]).a
```

Comment:
Test cases showing that `.Value` is a compiler error in these cases would also be good.

Reply:
Fair point